### PR TITLE
FizzBuzz Game

### PR DESCRIPTION
**Pull request by carolinestewart**
This is a very simple piece of code.   It has been added for use in testing and learning the functions of GitHub.

### CODE REVIEW

**Comment by RostiReadioff** [comment]
This is a very nice exercise. I suggest adding comments for the python codes.

File: Code/FizzBuzzGame/FizzBuzzInstructions.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+**_FizzBuzz Game_**  
+Displays numbers which divide by 3 and 5.
+
+
+**_Requirements_**  
+Anaconda
```
Comment (dblana) [inline Code/FizzBuzzGame/FizzBuzzInstructions.md]:
```suggestion
Python 3 
```

**Review by dblana — CHANGES_REQUESTED** [review]
This looks great! :sparkles: I've proposed a change (you don't necessarily need Anaconda to run this, just Python 3).

**Review by dblana — COMMENTED** [review]
More comments are always good! :smiley:

**Review by dblana — APPROVED** [review]
Well done! Happy to merge your files into CMAS-open-code :tada:

**Comment by carolinestewart** [comment]
Hi Dimitra,

I’ve done that now and added a comment as requested by Rosti.

**Pull request merged**